### ISSUE TITLE
sad to see that this aint working anymore

### ISSUE THREAD

**Issue by Phonfo** [Xeroday/Spotify-Ad-Blocker#259]
yup, this aint working and as i can see the devs doesnt support this anymore, i think its because of the new update of spotify that enhances security and protection to ad blocking programs

**Comment by PNSoniX** [comment]
Version 1.7.6 works for me. 1.8.0 and 1.8.1 dont. Just try 1.7.6

**Comment by TheDiamondGen** [comment]
Yeah 1.7.2 also works fine for me, I use 1.7.2 cuz higher versions break my spicetify theme for some reason.

**Comment by gmy77** [comment]
Yes very SAD... My God if i had at least a minimum of programming knowledge i would try ... every one in his profession ... and this unfortunately is not mine.
